src/libraries/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceFactory.byte.cs

```diff
@@ -37,7 +41,11 @@ internal class MemoryTestSequenceFactory : ReadOnlySequenceFactory<T>
         {
             public override ReadOnlySequence<T> CreateOfSize(int size)
             {
-                return CreateWithContent(new T[size]);
+#if DEBUG
+                return new ReadOnlySequence<T>(new ReadOnlyMemory<T>(new T[size + 1]).Slice(1)); // #57472
```

Comment:
I don't think we need to litter the code with references to issues (but if we do, it should be the full link rather than just a number).